### PR TITLE
More blog typical elements, pivot navigation

### PR DESCRIPTION
**Pull request by mariusschulz**
I added lists, sub headings, and a footer to the post sample. Also, the layout now includes a pivot-like navigation known from the Windows Phone.

### CODE REVIEW

**Comment by mariusschulz** [comment]
Pull request can't be auto-merged, I'll create a new one.

File: less/metro.less [hunk 1]
```diff
@@ -1,2 +1,15 @@
 @import "colours.less";
-@import "fonts.less";
\ No newline at end of file
+@import "fonts.less";
+
+#bundle {
+	.pivot-header {
+		color: @inactive;
+		font-size: 20px;
```
Comment (ghost) [inline less/metro.less:7]: We should make a note to put font-sizes in `fonts.less`, I suppose, for consistency?

File: less/metroblog.less [hunk 2]
```diff
@@ -0,0 +1,21 @@
+@import "metro.less";
+
+nav {
+	ul {
+		list-style-type: none;
+		padding-left: 0;
+
+		li {
+			display: inline;
+			margin-right: 20px;
```
Comment (ghost) [inline less/metroblog.less:10]: As above, should all margins go in a similar spot so we can get ratios working better - perhaps a `layout.less`?

File: less/metro.less (hunk 1 above)
Comment (mariusschulz) [inline less/metro.less:7]: Good point, the 20px are leftovers from testing.

File: less/metro.less [hunk 3]
```diff
@@ -1,2 +1,15 @@
 @import "colours.less";
-@import "fonts.less";
\ No newline at end of file
+@import "fonts.less";
+
+#bundle {
```
Comment (ghost) [inline less/metro.less:4]: Name should be more descriptive, can be changed later.

**Pull request merged**